runtime/lua/vim/lsp/util.lua

```diff
@@ -99,9 +99,26 @@ local function get_border_size(opts)
   return { height = height, width = width }
 end
 
-local function split_lines(value)
-  value = string.gsub(value, '\r\n?', '\n')
-  return split(value, '\n', { plain = true, trimempty = true })
```

Comment:
Why is this `trimempty` acceptable, but the new `split_lines` logic is not acceptable?

Reply:
Isn't that stuff we're trying to get rid of?